postprocessing/pp-opus.c

```diff
@@ -19,48 +19,42 @@
 #include <string.h>
 #include <stdlib.h>
 
-#include <ogg/ogg.h>
-
+#include "pp-avformat.h"
 #include "pp-opus.h"
 #include "pp-opus-silence.h"
 #include "../debug.h"
 #include "../version.h"
 
+static AVFormatContext *fctx;
+static AVStream *vStream;
 
-/* OGG/Opus helpers */
-FILE *ogg_file = NULL;
-ogg_stream_state *stream = NULL;
+static const uint8_t opus_extradata[19] = {
+    'O', 'p', 'u', 's', 'H', 'e', 'a', 'd',
+    1, 2, 56, 1, 128, 187,
+    0, 0, 0, 0, 0,
+};
```

Comment:
Whoops looks like I missed this space-indentation in the previous review, sorry...